spec/system/admin/import_rewards_from_csv_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe 'Import rewards from csv', type: :system do
+  let(:admin_user) { create(:admin_user) }
+  let(:category) { create(:category, title: 'Category 1') }
+  let!(:reward) { create(:reward, title: 'First reward', price: 1.0, category: category) }
+
+  before do
+    login_admin(admin_user)
+    visit('/admin/rewards')
+  end
+
+  context 'when importing rewards from CSV' do
+    it 'updates rewards and import new form uploaded file' do
+      click_link 'Import Rewards'
+      attach_file(File.absolute_path('./spec/fixtures/no_error.csv'))
```

Comment:
I believe it's better to do Rails.root.join('./spec/fixtures/no_error.csv'), but that's nitpick.
Also you could take advantage of Ruby not needing parenthesis so it's more readable like so

```suggestion
      attach_file Rails.root.join('./spec/fixtures/no_error.csv')
```